app/features/build/views/build.erb

```diff
@@ -34,11 +34,34 @@
 
 <hr />
 
+<% if build.artifacts.count > 0 %>
+  <h1>Artifacts</h1>
+  <table class="mdl-data-table mdl-js-data-table mdl-shadow--2dp">
+    <thead>
+      <tr>
+        <tr>
+          <th>Artifact type</th>
+          <th>Path</th>
+        </tr>
+      </tr>
+    </thead>
+    <tbody>
+      <% build.artifacts.each do |current_artifact| %>
+        <tr>
+          <td><%= current_artifact.type %></td>
+          <td><%= current_artifact.reference %></td>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+
+  <hr />
+<% end %>
 
 
 <code id="log">
-  <% if existing_rows.count > 0 %>
-    <%= existing_rows.join("\n") %>
+  <% if existing_rows.to_s.length > 0 %>
```

Comment:
Could we use [`positive?`](https://ruby-doc.org/core-2.3.0/Numeric.html#method-i-positive-3F) instead, please? I feel it's a bit more idiomatic ruby.

```ruby
if existing_rows.to_s.length.positive?
  # ...
end
```

Reply:
In this case, `existing_rows` isn't an array, but a string 👍 So we can't use `positive?`, which AFAIK would only work in combination with `.count`

Reply:
@KrauseFx 
```
irb(main):004:0> 0.positive?
=> false
irb(main):005:0> 1.positive?
=> true
irb(main):
```
I think we can use it 👍